src/infrastructure/config.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { afterAll, beforeAll, describe, expect, it } from 'vitest'
+import type { AppInstance } from '../app.js'
+import { getApp } from '../app.js'
+
+describe('config', () => {
+  let app: AppInstance
+
+  beforeAll(async () => {
+    app = await getApp({
+      monitoringEnabled: false,
+    })
+  })
+
+  afterAll(async () => {
+    await app.close()
+  })
+
+  it('support multi line values', () => {
+    const value = app.diContainer.cradle.config.multiLinevalue
```

Comment:
🔶  This file was removed in my previous PR

Reply:
ouch. good catch